src/app.ts

```diff
@@ -27,8 +26,7 @@ if (!process.env.PUBLIC_URI) {
 (async () => {
 
   console.log('Connecting to database');
-  await checkPatches();
-
+  console.log('Running Migrations');
```

Comment:
Is this a lie?

Reply:
Lol it happens when the pool connection is made but seemed weird with the logs going:

1. Connecting to database
2. Reading settings
3. Running Migrations

So I put it with the others, which are technically a lie too lol, I can move the console logs into the right places, I think thats the most correct

Reply:
Yeah can we? I imagine knex output will be interspersed with these too